packages/frontend/src/store/overtime.ts

```diff
@@ -1,96 +1,158 @@
 import { State } from "./index";
-import config from '@/config';
-import {adAuthenticatedFetch} from '@/services/auth';
-import {ActionContext} from 'vuex';
+import config from "@/config";
+import { adAuthenticatedFetch } from "@/services/auth";
+import { ActionContext } from "vuex";
 
 export interface OvertimeState {
-	overtimeState: OvertimeStateModel;
+  overtimeState: OvertimeStateModel;
 }
 
 export interface OvertimeStateModel {
-	availableHours: CompansatedTransactions[];
-	payoutTransactions: Transaction[];
-	flexTransactions: Transaction[];
-	totalHours: number;
-	compansatedHours: number;	
-	totalFlexedHours: number;
-	totalPayoutHours: number;
+  availableHours: CompansatedTransactions[];
+  payoutTransactions: PayoutTransaction[];
+  flexTransactions: FlexHoursTransaction[];
+  totalHours: number;
+  compansatedHours: number;
+  totalFlexedHours: number;
+  totalPayoutHours: number;
 }
 
 export interface CompansatedTransactions {
-		date: Date;
-		hours: number;
-		taskId: number;
-		compensationRate: number;
+  date: Date;
+  hours: number;
+  taskId: number;
+  compensationRate: number;
 }
 
 export interface AvailableHoursResponse {
-		totalHours: number;
-		totalHoursIncludingCompensationRate: number;
-		entries: CompansatedTransactions[];
+  availableHoursBeforeCompensation: number;
+  availableHoursAfterCompensation: number;
+  entries: CompansatedTransactions[];
+}
+
+export interface FlexHoursTransaction {
+  date: Date;
+  hours: number;
 }
 
 export interface Transaction {
```

Comment:
Kan vi gå igjennom det som har blitt gjort med Transaction + venner? Jeg tror transaction blir for generelt som navn.